lib/launcher.js

```diff
@@ -351,11 +351,16 @@ class Launcher {
 
     /**
      * emit event from child process to reporter
-     * @param  {Object} m  event object
+     * @param  {String} cid
+     * @param  {Object} m event object
      */
-    messageHandler (m) {
+    messageHandler (cid, m) {
```

Comment:
> Maybe we should get rid of `rid` in `startInstance` and `endHandler`

How do I do this, if in [`startInstance`](https://github.com/webdriverio/webdriverio/blob/master/lib/launcher.js#L292) we use both of these variables?